### PR TITLE
Merge @bitgo/utxo-lib into SDK monorepo

### PR DESCRIPTION
**Pull request by tylerlevine**
This will improve the workflow for releasing new changes to @bitgo/utxo-lib and aligns it with the single open source repo strategy that is employed by most of our other open source SDK packages.

All history from @bitgo/utxo-lib is preserved.

Since we have not been diligent in publishing only from the master branch, some wrangling was needed to get the @bitgo/utxo-lib repo in a state where it was mergeable. Specifically, the following procedure was used:

1) Clone existing bitgo-utxo-lib
1) Checkout https://github.com/BitGo/bitgo-utxo-lib/pull/89
1) Merge this into master locally
1) Create new branch off master in bitgo-utxo-lib repo (`BG-31305-merge-utxo-lib`)
1) Move all files into `modules/utxo-lib`
1) Add cloned bitgo-utxo-lib repo as remote repo of BitGoJS repo (using alias `utxo`)
1) Create new branch off master in BitGoJS repo (`BG-31305-merge-utxo-lib`)
1) `git merge --allow-unrelated-histories utxo/BG-31305-merge-utxo-lib`
1) Fix npm testing scripts, CODEOWNERS, package lock files, and disable one broken test

Ticket: BG-31305

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+sudo: false
```
Comment (ericcrosson-bitgo) [inline .travis.yml:1]: Is preserving this file intentional?

Comment (tylerlevine) [reply .travis.yml:1]: No, we can dump it

**Review by ericcrosson-bitgo — DISMISSED** [review]
LGTM :rocket: 

Only 2.2k commits to review

**Review by ericcrosson-bitgo — APPROVED** [review]
Still LGTM

**Pull request merged**